modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -89,7 +89,9 @@ def _unwrap_partitions(oid):
         ]
 
 
-def from_partitions(partitions, axis):
+def from_partitions(
```

Comment:
This should be changed as part of separate issue.

Reply:
This change will be moved to solution of #3338